CBLAS/cblas.pc.in

```diff
@@ -6,5 +6,5 @@ Description: C Standard Interface to BLAS Basic Linear Algebra Subprograms
 Version: @LAPACK_VERSION@
 URL: http://www.netlib.org/blas/#_cblas
 Libs: -L${libdir} -l@CBLASLIB@
+Libs.private: @BLAS_PCLIBS@
 Cflags: -I${includedir}
-Requires.private: @BLASLIB@
```

Comment:
`Requires` is better than `Libs.private` since it allows to inject a dependent pc file.

Reply:
For example. vcpkg builds lapack-reference with openblas. openblas installs a `openblas.pc` file. In vcpkg there is a metaport called blas which supposly selects the blas implementation. As such this metaport install a `blas.pc` files which simply has a `Requires(.private): openblas`.

Reply:
But with `Requires` you're stuck with pkg-config files. In a perfect world, you'd have .pc files for each installed BLAS and configuration of it. I'm working on that (although I see, from dealing with this year-old attempt at this stuff, that my motivation is waning towards more productive endeavours). But right now, the common denominator is a plain list of linker arguments, which you can easily provide using `pkg-config` in the cmake command line anyway.

You mention one way to deal with this mess by introducing a metaport that installs a redirecting `blas.pc`. Another layer of confusion … my world installs differing BLAS implementations at the same time and, while there will be a default one, not everyone is pointed to it. While this is a viable approach as such, we have the ambiguity here that the name `blas.pc` is traditionally taken by Netlib's implementation. If that is a placeholder, no actual imeplementation should install it.

There might be a discussion here … but in the context of a packaging system,  I don't see it, really. If you can put in
```
Requires.private: openblas
```
you can also expand that to what it would do, or even just copy the file.  Please understand that I'm not fighting the idea of having more things set up with `.pc` files, but that my current view is that things should work if the BLAS implementation doesn't offer one (or a broken one). BLAS doesn't usually depend on a lot of other libraries (well, usually), so the added value of a `.pc` file is limited. Just `Libs` is the more general solution.

Reply:
I'm leaning towards using the name netlib instead of reference. Somehow it seems to be more proper (traditional?).

About fixing up pkg-config files, I am actually intending to do that, including preparing one for Intel MKL stuff as used in the pkgsrc prefix. It's just that we have to live in a world with transitions. I'll focus now on getting the CMake BLAS stuff recognize given pkg-config files. Consequently, the netlib reference build should also just use pkg-config, in that world …  I just have to ponder a bit about actually ditching the tradition of providing `BLAS_LIBS` and `LAPACK_LIBS` with the actual linker arguments to use.